### DISCUSSION TITLE
Starting New PeerJs without port

### DISCUSSION THREAD

**Discussion by CaduDev** [peers/peerjs#1100] [Q&A]
I'm trying to create a new Peer(), and I have a server in node.js on render.com, it happens that the render doesn't provide the port, and PeerJs adds a port automatically if you don't pass this parameter

The question is:

How do I instantiate the Peer without the port?

**Comment by ddenizakpinar** [comment]
Could you find the answer?

**Comment by CaduDev** [comment]
unfortunately not yet, I tried everything and had no result, usually in development we pass the port but not in production, but I still haven't found a way to do this without using a port

**Comment by samhirtarif** [comment]
Out of curiosity, have you tried port `443`?

**Comment by ddenizakpinar** [comment]
I forgot to update here. @samhirtarif suggestion fixed the problem for me. Also I figured out that if you do not pass the port parameter it also uses the `443` by default would work without any problem on your example.com.

**Comment by CaduDev** [comment]
The port is encapsulated in the available domain, so if I pass the port it doesn't work, locally as I use localhos:port it works perfectly, but when it's in production it doesn't work

**Comment by jonasgloning** [comment]
I'm not sure what exactly the problem is. Please note that _every_ WebSocket/TCP connection has a port, you can not avoid setting one, and 443 is the default encrypted web port and a sane default. You _can_ override this if you want.

If you want to use the same port number as your current page, you could use something like
```ts
new Peer({ port: location.port })
```

Could you please provide additional details to better understand your situation: your goal, your code, your server URL, and your expected outcome?
